packages/web-client/app/components/card-pay/deposit-workflow/index.ts

```diff
@@ -52,22 +54,22 @@ class DepositWorkflow extends Workflow {
           componentName: 'card-pay/layer-one-connect-card',
         }),
       ],
-      completedDetail: 'Mainnet wallet connected',
+      completedDetail: `${capitalize(c.layer1.networkType)} wallet connected`,
```

Comment:
Wondering whether this should be the responsibility of the `NetworkCopywriting` type - capitalized vs non-capitalized stuff?

Reply:
Maybe… semi-relatedly, I anticipate some difficulty around things like “an Ethereum wallet” vs “a Kovan wallet” but I feel like it’s not worth doing anything in that direction until it‘s truly needed, it doesn’t seem like this implementation will hamper updates toward that end if a need does emerge.